command/kv_patch.go

```diff
@@ -120,7 +120,10 @@ func (c *KVPatchCommand) Run(args []string) int {
 	}
 
 	// First, do a read
+	curOutputCurl := client.OutputCurlString() // we don't want to see curl output for the read request
```

Comment:
I think this comment goes more with the subsequent line.  Also, I don't usually see comments on the same line after code in Vault; I don't think they're forbidden, but maybe discouraged?  I do know that I've been asked in previous PRs to ensure that comments stay within the first 80 columns, so maybe it's just that it's hard to have a useful comment that fits that constraint and isn't on its own line.

Reply:
fixed